models/google_play__country_report.sql

```diff
@@ -0,0 +1,150 @@
+with installs as (
+
+    select *
+    from {{ var('stats_installs_country') }}
+
+), ratings as (
+
+    select *
+    from {{ var('stats_ratings_country') }}
+
+
+), store_performance as (
```

Comment:
Can you update the CTEs to be in line with our new [style guidelines](https://github.com/fivetran/dbt_style_guide/blob/main/sql_guidelines.md#ctes).